calico_containers/calico_ctl/node.py

```diff
@@ -191,12 +195,13 @@ def node(arguments):
                    as_num=as_num,
                    detach=detach,
                    kubernetes=arguments.get("--kubernetes"),
+                   kubernetes_version=kubernetes_version,
                    rkt=arguments.get("--rkt"),
                    libnetwork_image=libnetwork_image)
 
 
-def node_start(node_image, log_dir, ip, ip6, as_num, detach, kubernetes, rkt,
-               libnetwork_image):
+def node_start(node_image, log_dir, ip, ip6, as_num, detach, kubernetes,
```

Comment:
Don't need separate kubernetes boolean as version will be None if kubernetes is False.